webbpsf/optics.py

```diff
@@ -727,3 +727,43 @@ def _calc_blc_wedge(deg=4, wavelength=2.1e-6):
 
 
 
+# Field dependent aberration class for JWST instruments
+
+class JWST_Field_Dependent_Aberration(poppy.OpticalElement):
+    """ Field dependent aberration generated from Zernikes measured in ISIM CV testing
```

Comment:
This class obviously isn't intended to work yet, it just got swept up along with everything else on this branch in the PR.